ChildProject/pipelines/samplers.py

```diff
@@ -357,6 +357,10 @@ def _segment_scores(self, recording):
         segments = self.retrieve_segments(recording['recording_filename'])
 
         if segments is None:
+            print("warning: no annotations from the set '{}' were found for the recording '{}'".format(
+                self.annotation_set,
+                recording['recording_filename']
+            ))
             return pd.DataFrame(columns = ['segment_onset', 'segment_offset', 'recording_filename'])
 
         # NOTE: The timestamps were simply rounded to 5 minutes in the original 
```

Comment:
what is the alternative here?
I should mention that LENA users typically have time-stamped audios that make reference to local time, so they'll use 5 min chunks of clock time; VTC users may not always have this (at least right now), so we tend to do time from the onset of the file.

Reply:
Just to clarify: 

We have made the length of the clips a parameter, so any value can be used (30 s, 1 minute, 5 minutes, etc.)

But I think we should add offset parameter to all window-based samplers, so that people can align them as they wish (e.g. beginning at a specific clocktime). This is also useful when you need to align windows for simultaneous recordings from different devices.

I've done that for the PeriodicSampler and the EnergySampler.